packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -263,13 +280,25 @@ export default function SpanDetail(props: SpanDetailProps) {
           />
         )}
         <small className={styles.debugInfo}>
+          <a
```

Comment:
Currently, this is an anchor tag that on click, will focus the span. To share a link I would right click -> copy url.

I've gotten feedback that users would prefer a click-to-copy button, so I may want to update this behavior